### ISSUE TITLE
[Windows] No color preview in gutter, even with the colorDecorators option set

### ISSUE THREAD

**Issue by navaronbracke** [Dart-Code/Dart-Code#4874]
**Describe the bug**
Color previews don't show up on Windows.

**To Reproduce**
1. `flutter create bug`
2. Add `const Color c = Color(0xFFFFFFFF);` in the build method of the `MyApp` widget
3. Add `"editor.colorDecorators": true` to the workspace settings
4. See no color previews in the gutter.

**Expected behavior**
I expect the color preview to show up in the gutter, like it does on MacOS.

This related issue was closed: https://github.com/Dart-Code/Dart-Code/issues/3925

**Screenshots**
![image](https://github.com/Dart-Code/Dart-Code/assets/23480364/e4d78601-7237-4bd7-a633-b4319693ad96)

```
[√] Flutter (Channel stable, 3.16.0, on Microsoft Windows [Version 10.0.19045.3693], locale nl-BE)
    • Flutter version 3.16.0 on channel stable at E:\Documenten\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision db7ef5bf9f (3 weeks ago), 2023-11-15 11:25:44 -0800
    • Engine revision 74d16627b9
    • Dart version 3.2.0
    • DevTools version 2.28.2

[√] Windows Version (Installed version of Windows is version 10 or higher)

[√] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at E:\Android\Sdk
    • Platform android-34, build-tools 34.0.0
    • ANDROID_HOME = E:\Android\Sdk
    • Java binary at: E:\Documenten\android-studio-hedgehog\android-studio\jbr\bin\java
    • Java version OpenJDK Runtime Environment (build 17.0.7+0-b2043.56-10550314)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files (x86)\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop Windows apps (Visual Studio Community 2022 17.7.6)
    • Visual Studio at E:\Program Files\VisualStudio
    • Visual Studio Community 2022 version 17.7.34221.43
    • Windows 10 SDK version 10.0.22621.0

[√] Android Studio (version 2023.1)
    • Android Studio at E:\Documenten\android-studio-hedgehog\android-studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.7+0-b2043.56-10550314)

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.19045.3693]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 103.0.5060.134
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 119.0.2151.93

[√] Network resources
    • All expected network resources are available.

• No issues found!
```

**Comment by DanTup** [comment]
The gutter colour previews are a legacy feature that was removed after we added support for colours that is built-in to VS Code + LSP. These appear inline and show a colour picker:

![image](https://github.com/Dart-Code/Dart-Code/assets/1078012/3a5dea00-7cbf-4ef3-afed-d80d1952e342)

Have you restarted VS Code since `"editor.colorDecorators": true` was set, as I'm not sure if applies immediately?


> I expect the color preview to show up in the gutter, like it does on MacOS.

If you're still seeing them in the gutter on macOS, I suspect you're on an old version of the Dart extension. The code to show them in the gutters was removed in https://github.com/Dart-Code/Dart-Code/commit/423fa8da6a6576343148253289bc22b58d5fce2c.
